src/robot/libraries/Process.py

```diff
@@ -349,26 +349,38 @@ def run_process(self, command, *arguments, **configuration):
         Note that possible equal signs in ``*arguments`` must be escaped
         with a backslash (e.g. ``name\\=value``) to avoid them to be passed in
         as ``**configuration``.
+        
+        ``log_level`` value can be ``info``, ``debug``, ``warn`` and ``error``.
+        If no value provided, then all log will be info type.
 
         Examples:
         | ${result} = | Run Process | python | -c | print('Hello, world!') |
         | Should Be Equal | ${result.stdout} | Hello, world! |
         | ${result} = | Run Process | ${command} | stdout=${CURDIR}/stdout.txt | stderr=STDOUT |
         | ${result} = | Run Process | ${command} | timeout=1min | on_timeout=continue |
         | ${result} = | Run Process | java -Dname\\=value Example | shell=True | cwd=${EXAMPLE} |
+        
+        ``log_level`` examples:
+        | ${result} = | Run Process | python | 'INFO' | -c | print('Hello, world!') |
+        | Should Be Equal | ${result.stdout} | Hello, world! |
+        | ${result} = | Run Process | ${command} | 'debug' | stdout=${CURDIR}/stdout.txt | stderr=STDOUT |
+        | ${result} = | Run Process | ${command} | 'Warn' | timeout=1min | on_timeout=continue |
+        | ${result} = | Run Process | java -Dname\\=value Example | 'Debug' | shell=True | cwd=${EXAMPLE} |
```

Comment:
There's no need to duplicate all examples with `log_level`. Adding a new example with it, or possibly just adding it to one of the existing examples, is enough.